README.md

```diff
@@ -7,7 +7,7 @@ A GitHub bot that rebases your pull request branches when you ask
 1. Make a dedicated GitHub account for the bot
 2. Grant the GitHub account read and write access to your repositories
 3. [Setup](#setup) the bot on your own server
-4. **@{github bot username} rebase** the bot in a pull request comment
+4. Type a comment **@{github bot username} rebase** in a pull request
```

Comment:
How about wrapping it with quotes as well?

Example:
Type a comment "**@{github bot username} rebase**" in a pull request